hack/install.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/sh
+
+set -eux
+
+# Required operator-sdk version
+REQUIRED_OPERATOR_SDK_VERSION="v0.12.0"
+
+if [ ! "$(command -v operator-sdk)" ]; then 
+    LOCAL_OPERATOR_SDK_VERSION=""
+else 
+    LOCAL_OPERATOR_SDK_VERSION="$(operator-sdk version | head -n1 | cut -d" " -f3 | tr -d '",')"
+fi
+
+# If local operator-sdk version is not required version
+# or operator-sdk is not installed, install operator-sdk v0.12.0
+if [ "${LOCAL_OPERATOR_SDK_VERSION}" != "${REQUIRED_OPERATOR_SDK_VERSION}" ] || [ -z "${LOCAL_OPERATOR_SDK_VERSION}" ]; then
+    mkdir -p ./bin
```

Comment:
Should we create `.bin` folder here? With this command, the new folder will be added into `git status` command after that.
How about  `$GOPATH/bin` folder?